src/services/CertificateGenerationService.ts

```diff
@@ -472,7 +472,7 @@ class CertificateGenerationService {
         ) => {
           const payload: any =
             this.lambdaClient.validateInvocationResponse(response);
-          let testResults: any[] = JSON.parse(payload.body);
+          const testResults: any[] = JSON.parse(payload.body);
```

Comment:
We can update this to correct type `ITestResult` instead of `any`.

Reply:
I did look at updating this. However the ITestResult used in cert-gen differs from ITestResult returned by the test-results service. The cert-gen interface maps testTypes to an object, but the test-results interface is actually returning an array. 

If I change the ITestResult interface, then that may impact other methods which are currently using it. ie - generateCertificateData method.

I thought maybe this could be cleaned up via an improvement story?